server-cloud/server/server.go

```diff
@@ -91,14 +121,22 @@ func (s *Server) Serve(ctx context.Context, port int) error {
 	p := prometheus.NewPrometheus("echo", nil)
 	p.Use(e)
 
-	// Add other routes here...
+	gob.Register(map[string]interface{}{})
```

Comment:
Why is this line necessary? If it's for cookie serialization, consider serializing as JSON instead.